reactions.json

```diff
@@ -190,14 +202,18 @@
     "Br2 / CCl4": "4-bromophenol",
     "conc. HNO3": "2,4,6-trinitrophenol",
     "CO2 / NaOH / H+ (423K, 3-7 atm)": "2-hydroxybenzoic acid",
-    "H2SO4/ (373K)": "4-hydroxysulphonic acid"
+    "H2SO4/ (373K)": "4-hydroxysulphonic acid",
+    "NaOH / CO2 / H+": "2-hydroxybenzoic acid",

```

Comment:
This is the same as the reaction in line 204

Reply:
unusre about the name